pkg/resourcegroup/runaway/record.go

```diff
@@ -461,3 +461,23 @@ func (rm *Manager) RemoveRunawayWatch(recordID int64) error {
 	err = handleRunawayWatchDone(rm.sysSessionPool, records[0])
 	return err
 }
+
+// RemoveRunawayResourceGroupWatch is used to remove all runaway watch items of a resource group.
+func (rm *Manager) RemoveRunawayResourceGroupWatch(groupName string) error {
+	rm.runawaySyncer.mu.Lock()
+	defer rm.runawaySyncer.mu.Unlock()
+	records, err := rm.runawaySyncer.getWatchRecordByGroup(groupName)
+	if err != nil {
+		return errors.Annotate(err, "get watch records by resource group failed")
+	}
+	if len(records) == 0 {
+		return errors.Errorf("no runaway watch found for resource group: %s", groupName)
```

Comment:
I prefer to return nil when there is no watch records as it's similar to delete statement.